src/cloud_common/protocols/unit/unit_config.py

```diff
@@ -172,11 +172,11 @@ class AlertRules(BaseModel):
         ),
     ]
 
-    used_disks: Annotated[
+    disks: Annotated[
         Optional[list[AlertRulePercentsOfDisk]],
         Field(
             default=None,
-            alias='usedDisks',
+            alias='disks',
```

Comment:
`disk`

Reply:
why 'disk' if it is a list?

Reply:
done. disks -> disk

Reply:
Cloud can support both `disk` and `disks` for backward compatibility